pkg/rtc/audiolevel.go

```diff
@@ -7,57 +7,57 @@ import (
 
 const (
 	// number of audio frames for observe window
-	observeFrames    = 25 // webrtc default opus frame size 20ms, 25*20=500ms matching default UpdateInterval
+	observeDuration  = 500 // 500ms matching default UpdateInterval
```

Comment:
can leave out `matching default UpdateInterval` as that is not relevant in this time based approach.